### ISSUE TITLE
Add basic sleep parameters

### ISSUE THREAD

**Issue by Pugliane** [ghammad/pyActigraphy#139]
Thank you for the amazing package!
Could you please add a code for the analysis of the basic sleep parameters: bed time, get up time, time in bed, total sleep time, sleep onset and offset, efficiency of sleep, sleep latency onset, number of awakenings and WASO.

**Comment by islahtk** [comment]
Yes, it will be amazing to have those computed with different chosen algorithms. Or is there any way to compute them using parameters generated from this package?
